MapboxSearch/offline/src/androidTest/java/com/mapbox/search/offline/OfflineSearchEngineIntegrationTest.kt

```diff
@@ -646,18 +647,19 @@ internal class OfflineSearchEngineIntegrationTest {
                     houseNumber = "2011",
                     street = "15th Street Northwest",
                     // See SWEB-1113 for more history
-                    region = createCoreSearchAddressRegion("District of Columbia")
+                    region = createCoreSearchAddressRegion("District of Columbia"),
+                    country = createCoreSearchAddressCountry("United States")
                 )
             ),
             // See SWEB-1113 for more history
-            descriptionAddress = "2011 15th Street Northwest, District of Columbia",
+            descriptionAddress = "2011 15th Street Northwest, District of Columbia, United States",
 
             distanceMeters = 0.0,
             center = Point.fromLngLat(-77.03404491238354, 38.9179071535832),
             serverIndex = null,
         )
 
-        const val DOUBLE_COMPARISON_EPS = 0.00001
+        const val DOUBLE_COMPARISON_EPS = 0.0001
```

Comment:
Maybe worth putting a comment about the number of meters it's roughly translated too...

Reply:
added in 2c965427